cmd/portal/main.go

```diff
@@ -15,58 +14,40 @@ import (
 // injected at link time using -ldflags.
 var version string
 
-// Initialization of cobra and viper.
-func init() {
-	initConfig()
-	setDefaults()
+// -------------------------------------------------------- Root -------------------------------------------------------
 
+func Root() (*cobra.Command, error) {
+	if err := config.Init(); err != nil {
+		return nil, fmt.Errorf("initialising config: %w", err)
```

Comment:
initialising -> initializing (to keep consistent with American English spelling)